src/zope/testing/renormalizing.txt

```diff
@@ -234,3 +234,59 @@ Combining a checker with something else does not work:
         ...
     TypeError: unsupported operand type(s) for +: ...
 
+Using the 2to3 exception normalization:
+
+    >>> from zope.testing.renormalizing import EXCEPTION_2TO3
+    >>> checker = OutputChecker()
+    >>> want = """\
+    ... Traceback (most recent call last):
+    ... foo.bar.FooBarError: requires at least one argument."""
+    >>> got = """\
+    ... Traceback (most recent call last):
+    ... FooBarError: requires at least one argument."""
+    >>> result = checker.check_output(want, got, EXCEPTION_2TO3)
+    >>> import sys
+    >>> if sys.version < '3':
+    ...     expected = True
+    ... else:
+    ...     expected = False
+    >>> result == expected
+    True
+
+When reporting a failing test and running in Python 2, the normalizer tries
+to be helpful by explaining how to test for exceptions in the traceback output.
+
+    >>> want = """\
+    ... Traceback (most recent call last):
+    ... foo.bar.FooBarErrorXX: requires at least one argument.
+    ... """
+    >>> got = """\
+    ... Traceback (most recent call last):
+    ... FooBarError: requires at least one argument.
+    ... """
+    >>> checker.check_output(want, got, EXCEPTION_2TO3)
+    False
+    >>> from doctest import Example
+    >>> example = Example('dummy', want)
+    >>> result = checker.output_difference(example, got, EXCEPTION_2TO3)
+    >>> output = """\
+    ... Expected:
+    ...     Traceback (most recent call last):
+    ...     foo.bar.FooBarErrorXX: requires at least one argument.
+    ... Got:
+    ...     Traceback (most recent call last):
+    ...     FooBarError: requires at least one argument.
+    ... """
+    >>> hint = """\
+    ...     ===============================================================
+    ...     HINT:
+    ...       You seem to test traceback output.
+    ...       The optionflag EXCEPTION_2TO3 is set.
+    ...       Do you use the full dotted name for the exception class name?
```

Comment:
This hint is not helpful.  Is it saying that I should use the full dotted name, or that I shouldn't?

I'm starting to think that instead of `# doctest: +EXCEPTION2TO3` I want `#doctest: +IGNORE_EXCEPTION_MODULE`, and then do the stripping on both the `want` and the `got` texts, irrespective of the current Python version.